src/pages/homepage.jsx

```diff
@@ -26,6 +27,10 @@ videoState.liked.some((item) => item._id === video._id) ?
 removeLikes(token, video._id) : getLikes(token, video)
 }
 
+const playVideoHandler = (token, video) =>{
+    getHistory(token, video)
+    
+}
```

Comment:
I think this is a reduntant function as I dont see it getting called anywhere, Can be removed?

Reply:
Oh yes, forgot to remove it.